extensionsGallery-insider.json

```diff
@@ -4507,6 +4507,67 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "88",
+					"extensionName": "lz-scripts",
+					"displayName": "Lazy Scripts",
+					"shortDescription": "Generates scripts from the results of select query",
+					"publisher": {
+						"displayName": "Faith S. Yintii",
+						"publisherId": "LycanII",
+						"publisherName": "LycanII"
+					},
+					"versions": [
+						{
+							"version": "0.0.1",
+							"lastUpdated": "27/02/2022",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
```

Comment:
For 3rd party extensions we don't support direct links to the VSIX unfortunately - you will need to use the `Microsoft.SQLOps.DownloadPage` assetType and provide a link to the download page (e.g. https://github.com/LycanII/LzScripts/releases/tag/Latest for yours)

Reply:
Done please